src/Data/SOAS.hs

```diff
@@ -661,3 +628,31 @@ concatNameBinderLists :: Foil.NameBinderList i l -> Foil.NameBinderList n i -> F
 concatNameBinderLists lst Foil.NameBinderListEmpty = lst
 concatNameBinderLists lst (Foil.NameBinderListCons x xs) =
   Foil.NameBinderListCons x (concatNameBinderLists lst xs)
+
+-- swapSum
+--   :: Sum sig1 sig2 scope term
+--   -> Sum sig2 sig1 scope term
+-- swapSum (L2 x) = R2 x
+-- swapSum (R2 y) = L2 y
+
+-- swapAnnSum
+--   :: AnnSig t (Sum sig1 sig2) scope term
+--   -> AnnSig t (Sum sig2 sig1) scope term
+-- swapAnnSum (AnnSig sig t) = AnnSig (swapSum sig) t
+
+-- transAST
+--   :: (Bifunctor sig)
+--   => (forall a b. sig a b -> sig' a b)
+--   -> AST binder sig n
+--   -> AST binder sig' n
+-- transAST _phi (Var x) = Var x
+-- transAST phi (Node node) =
+--   Node (phi (bimap (transScopedAST phi) (transAST phi) node))
+
+-- transScopedAST
+--   :: (Bifunctor sig)
+--   => (forall a b. sig a b -> sig' a b)
+--   -> ScopedAST binder sig n
+--   -> ScopedAST binder sig' n
+-- transScopedAST phi (ScopedAST binder body) =
+--   ScopedAST binder (transAST phi body)
```

Comment:
I think code should just be removed. if it's ever needed again, it's still in git history